### PR TITLE
Handle calculator events

### PR DESCRIPTION
**Pull request by Maha-Magdy**
Hello 😊,
In this pull request, I have made the following:-
- Copy the files calculate.js and operate.js into a logic/ directory in my project.
- Analyze the files calculate.js and operate.js, be sure I understand what they do.
- Implement the event handlers I need to use the math logic from calculate.js and operate.js in my React component.
- Practice what I have learned until now: use state and props, pass props to child components, use lifecycle methods, lift the state up.
- Test my app in the browser. It is able to perform math operations.

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -5,36 +5,36 @@ export default class Calculator extends React.Component {
   render() {
```
Comment (eliasecasta) [inline src/components/Calculator.js:39]: - [x] The Calculator component should import the calculate.js file from the logic directory, this means that your logic will need to change a little bit to adapt to this change. You will most likely need to move some content from your App.js script to the Calculator.js component, please make these changes to comply with the project requirements.

File: src/components/Calculator.js [hunk 2]
```diff
@@ -5,36 +5,36 @@ export default class Calculator extends React.Component {
...
     return (
       <div className="calculator-container">
-        <input type="text" placeholder="0"></input>
+        <input type="text" value={this.props.CalculatorScreen}></input>
         <table>
           <tr>
-            <td className="bg-light-gray">AC</td>
```
Comment (eliasecasta) [inline src/components/Calculator.js:11]: - [x] Your AC button is not working, please take a look at it. (It is not removing the current value from the calculator screen)

**Review by eliasecasta — CHANGES_REQUESTED** [review]
# Status:  Changes Required ♻️
_Check the comments under the review_
<hr>

Hello @Maha-Magdy, I am Elias, your reviewer for this PR

Well done so far with your project!

Almost all requirements are done in this project, you are just missing two really small ones.

I will mention to you a small summary of things that need to be changed in order to keep moving:

- [x] Your AC button is not working, please take a look at it.
- [x] You need to import the calculate.js file inside your calculator component to comply with the project requirements, please do it.
- [x] Please take a look at the comments below.

Please correct those issues and send another review request, good luck, and well done so far! 

Code review by Elias Castañeda

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear)_

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

As described in the [Code reviews limit Policy](https://microverse.zendesk.com/hc/en-us/restricted?return_to=https%3A%2F%2Fmicroverse.zendesk.com%2Fhc%2Fen-us%2Farticles%2F1500004088561) you have a limited number of code reviews per project (you can check the exact number in your Dashboard). If you think that the code review was not fair, you can [request a second opinion using this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**